test/module/irohad/validation/stateful_validator_test.cpp

```diff
@@ -147,19 +147,30 @@ class Validator : public testing::Test {
  * @then all of them will appear in verified proposal @and errors will be empty
  */
 TEST_F(Validator, AllTxsValid) {
-  auto tx = TestTransactionBuilder()
-                .creatorAccountId("doge@master")
-                .createdTime(iroha::time::now())
-                .quorum(1)
-                .createAsset("doge", "coin", 1)
-                .build();
-  auto proposal =
-      TestProposalBuilder()
-          .createdTime(iroha::time::now())
-          .height(3)
-          .transactions(
-              std::vector<shared_model::proto::Transaction>{tx, tx, tx})
-          .build();
+  std::vector<shared_model::proto::Transaction> txs;
```

Comment:
You could void the boilerplate like this
```suggestion
  std::vector<shared_model::proto::Transaction> txs(
      3,
      TestTransactionBuilder()
          .creatorAccountId("doge@master")
          .createdTime(iroha::time::now())
          .quorum(1)
          .createAsset("doge", "coin", 1)
          .build());
```

Reply:
This will invoke copy ctor:
```C++
vector(size_type count,
       const T& value,
       const Allocator& alloc = Allocator());
```

Reply:
Is there a reason we should avoid copying? The original test (before this refactoring) did copy the tx: https://github.com/hyperledger/iroha/blob/ec3e0ca52e248c1ed6fbb55b5f4db87a97b960bb/test/module/irohad/validation/stateful_validator_test.cpp#L160-L161

Reply:
I see, if you want to avoid using copy ctor, you may use a lambda that generates that tx, an call it several times where needed.